src/Gui/Application.cpp

```diff
@@ -2001,10 +2001,9 @@ bool onlySingleInstance(GUISingleApplication& mainApp)
                 fn = QDir::cleanPath(fn);
             }
 
-            QByteArray msg = fn.toUtf8();
-            msg.prepend("OpenFile:");
-            if (!mainApp.sendMessage(msg)) {
-                qWarning("Failed to send message to server");
+            fn.prepend(QString::fromStdString("OpenFile:"));
```

Comment:
No reason to roundtrip through `std::string` here, right? You could use `QLatin1StringView` or one of the other related classes.

Reply:
No, indeed, I used `QLatin1String` because it works for both Qt 5 and 6.